state/open.go

```diff
@@ -549,7 +553,7 @@ func newState(
 		st.policy = newPolicy(st)
 	}
 	// Record this State instance with the global tracker.
-	GlobalTracker.Add(st)
+	profileTracker.Add(st, 1)
```

Comment:
I guess we aren't worried about the fact that this keeps a reference to the state? If the state isn't closed the worker goroutines will keep it alive anyway.